rust/agama-software/src/zypp_server.rs

```diff
@@ -330,6 +346,7 @@ impl ZyppServer {
             }
         }
 
+        _ = progress.cast(progress::message::Next::new(Scope::Software));
```

Comment:
Should we report communication errors?


```suggestion
        progress.cast(progress::message::Next::new(Scope::Software))?;
```